### PR TITLE
Added Selenium and improved configuration regarding to changes after …

### PR DESCRIPTION
**Pull request by partikus**
…first meeting

@php-refactor

### CODE REVIEW

**Comment by partikus** [comment]
@dominikhajduk I had run few machines so that's way I changed IP address, of course I'll revert that,

btw. next time please comment the PR instead the single commit as you did, thx

File: Vagrantfile [hunk 1]
```diff
@@ -3,15 +3,16 @@ app_config = YAML::load_file("./.project_name.yml")['parameters'];
 
 Vagrant.configure("2") do |config|
   config.vm.box = "ubuntu/trusty64"
-  config.vm.network :private_network, ip: "10.0.0.200"
+  config.vm.network :private_network, ip: "10.0.0.208"
```
Comment (dominikhajduk) [inline Vagrantfile]: @partikus true my mistake. This comment should be in right place..

Please revert IP to `.200`. Thanks.

@php-refactor/team-member any other comments?

**Comment by partikus** [comment]
@dominikhajduk done, @php-refactor ping

**Comment by dominikhajduk** [comment]
:+1

**Comment by partikus** [comment]
@dominikhajduk ping @php-refactor

**Pull request merged**